docset/winserver2025-ps/smbshare/New-SmbServerAlternativePort.md

```diff
@@ -0,0 +1,226 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: SmbServerAlternativePort.cdxml-help.xml
+Module Name: SmbShare
+ms.date: 02/22/2024
+online version: https://learn.microsoft.com/powershell/module/smbshare/new-smbserveralternativeport?view=windowsserver2025-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: New-SmbServerAlternativePort
+---
+
+# New-SmbServerAlternativePort
+
+## SYNOPSIS
+Creates a new alternative port for the Server Message Block (SMB) protocol on the local server.
+
+## SYNTAX
+
+```
+New-SmbServerAlternativePort [-TransportType] <TransportType> [-Port] <UInt16>
+ [-EnableInstances] <Instance> [-Force] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>]
+ [-AsJob] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `New-SmbServerAlternativePort` cmdlet creates a new alternative port for the Server Message
+Block (SMB) protocol on the local server. This cmdlet can be used to add a new port to the list of
+alternate ports that are available for SMB connections.
+
+## EXAMPLES
+
+### Example 1: Create a new SMB server alternative port
+
+```powershell
+PS C:\\> New-SmbServerAlternativePort -TransportType QUIC -Port 1
+```
+
+This command creates a new SMB server alternative port that uses TCP transport on port number
+**1**.
+
+## PARAMETERS
+
+### -AsJob
+
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to
+complete.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -CimSession
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](https://go.microsoft.com/fwlink/p/?LinkId=227967)
+or [Get-CimSession](/powershell/module/cimcmdlets/get-cimsession) cmdlet. The default is the
+current session on the local computer.
+
+```yaml
+Type: CimSession[]
+Parameter Sets: (All)
+Aliases: Session
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnableInstances
+
+Enables the creation of multiple instances of the SMB server service, each with its own unique port
```

Comment:
Enables alternative ports on SMB instances. Currently only the Default instance is allowed, so this setting is not used.